bedrock/products/templates/products/vpn/more/base-article.html

```diff
@@ -33,6 +21,7 @@
       desc=self.page_desc()
     ) %}
     <div class="vpn-hero-cta">
+    HELLO
```

Comment:
Looks like a leftover from local testing that needs to be removed.

Reply:
Looking further, I'm not even sure this template is still in use? Which I'm guessing is why you added the HELLO here. The /vpn/more/* pages all seem to redirect to the resource center now, so it seems like we can clean up these templates as well. That could also be done in a follow-up PR rather than piling into this one.

Reply:
Ha, yes I was trying to get the page to figure out what the hell uses this template. Turns out they are still used / live URLs.

I filed https://github.com/mozilla/bedrock/issues/15245 to clean some of the oldest templates up.